FuncGodot Manual/pages/tips_runtime_building.html

```diff
@@ -0,0 +1,134 @@
+        <p>If using <a href="https://docs.godotengine.org/en/4.2/tutorials/3d/global_illumination/using_lightmap_gi.html" target="_blank">LightmapGI</a> for your global illumination solution, you can use the <i>FuncGodotMap</i> node's <code>unwrap_uv2()</code> method after receiving the <code>build_complete</code> signal. Once unwrapping has finished, the <i>FuncGodotMap</i> node will emit <code>unwrap_uv2_complete</code>.</p>
+        <blockquote><i>NOTE: Godot 4.2 does not currently support runtime lightmap baking or UV2 unwrapping in exported builds. For now, consider using a different lighting solution for runtime building.<br></i></blockquote>
+        An example of runtime map building is as follows:</p>
+        <pre><code>extends Node
+
+@onready var func_godot_map := $FuncGodotMap as FuncGodotMap
+
+# We can connect the various build signals of the FuncGodotMap 
+# in order to add more functionality after building.
+func _ready():
+    func_godot_map.connect("build_complete", _build_complete)
+    func_godot_map.connect("build_failed", _build_failed)
+    func_godot_map.connect("unwrap_uv2_complete", _unwrap_uv2_complete)
+    func_godot_map.local_map_file = "res://maps/runtime_building.map"
+    func_godot_map.verify_and_build()
+
+# We need to wait for the build to finish before unwrapping mesh UV2s for lightmap baking.
+# Currently lightmap baking cannot be performed at runtime, so it's not terribly useful yet.
+# Consider creating a procedural Voxel GI solid class entity that can be baked at runtime instead.
+func _build_complete() -> void:
+    print("Success! Unwrapping UV2...")
+    func_godot_map.unwrap_uv2()
+
+func _build_failed() -> void:
+    printerr("Failed to build the map file! :(")
+
+func _unwrap_uv2_complete() -> void:
+    print("UV2 unwrapping completed!")</code></pre>
+    <p>An example of this can be found in <a href="https://github.com/func-godot/func_godot_runtime_building" target="_blank">func_godot_runtime_building repository</a>.</p>
+
+    <h1>Building from Map Editors</h1>
+    <p>
+        Much like idTech engine games, it's possible to build and launch directly into a map from NetRadiant Custom and TrenchBroom alike.
+        This can be quite a helpful tool for mappers to quickly try out their creations without a lengthy startup or debug menu process.
+    </p>
+
+    <h2>Godot project setup example</h2>
+
+    <p>
+        Regardless of which map editor you use, you'll need to check for any 
+        <a href="https://docs.godotengine.org/en/stable/classes/class_os.html#class-os-method-get-cmdline-user-args">command line arguments.</a>
+
+        <p align="center"><img src="../images/tips_editorbuild_1.png"><br></p>
+
+        A basic example might look something like this:
```

Comment:
Weird phrasing, especially when omitting the "NetRadiant Custom and Trenchbroom" bit from earlier. We also don't need the image of the docs page, just the link, especially since we have an example of possible usage that does better to illustrate how to pull it off.
```html
<p>To build and launch our maps from the editor, you'll need to employ the <a href="https://docs.godotengine.org/en/stable/classes/class_os.html#class-os-method-get-cmdline-user-args" target="_blank">OS class' ability to retrieve command line arguments</a>. A basic example might look something like this:</p>
```